### DISCUSSION TITLE
Import external tiles

### DISCUSSION THREAD

**Discussion by shenshengyi** [iTwin/itwinjs-core#1402] [Q&A] [answered]
hi.I can use addTiledGraphicsProvider to import simple geometric elements created with Microstation and see them in the view, but after importing more complex geometric elements created by ORD, they cannot be seen in the view.
I would like to ask, is there a limit to the tiles imported using addTiledGraphicsProvider? Thanks.

**Comment by pmconne** [comment, marked as answer]
No.
For further assistance provide further information.
